torch/_inductor/compile_fx.py

```diff
@@ -489,6 +492,8 @@ def compile_fx_inner(
     if static_input_idxs is None:
         static_input_idxs = []
 
+    static_inputs_log.debug("static input idxs compile_fx_inner: %s", static_input_idxs)
```

Comment:
Can we get this in tlparse too? (it will probably make debugging why cudagraphs aren't running easier on internal stuff)

Reply:
(I'm not actually super familiar but I see some examples using `trace_structured` here https://github.com/pytorch/pytorch/blob/main/torch/_inductor/compile_fx.py#L728)